charts/monasca/templates/client-deployment.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.client.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "client.fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Values.client.name }}-deployment"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        component: "{{ .Values.client.name }}-deployment"
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+        - name: {{ template "name" . }}-{{ .Values.client.name }}-deployment
+          image: "{{ .Values.client.image.repository }}:{{ .Values.client.image.tag }}"
+          imagePullPolicy: {{ .Values.client.image.pullPolicy }}
+          command: ["sleep", "infinity"]
+          env:
+            - name: OS_AUTH_URL
+              value: "http://{{ template "keystone.fullname" . }}:{{ .Values.keystone.service.port }}/v3"
+            - name: OS_USERNAME
+              value: {{ .Values.client.keystone.os_username | quote }}
```

Comment:
should we grab this from keystone values? so we only have to set once? Eventually i want to have one section for keystone/database credientals? Not a huge deal we could also address down the road as I know i did the same in some places :)